### DISCUSSION TITLE
Scaling Axis Map

### DISCUSSION THREAD

**Discussion by kieranhiller** [mtex-toolbox/mtex#1399] [Ask Anything]
Hi,

My code in question is the following:

v = axis(ebsd('indexed').orientations,oriRef);
plot(v,'add2all','MarkerSize',1,'MarkerEdgeColor','black')

This plots points for the axis of orientation of each pattern in my data relative to the reference orientation, oriRef.. In the image I've attached you can see the radius of the circle is ~1.4degrees.

My question is how do I change this limit, i.e. say I want the radius of the circle the points are plotted on to be 45 degrees or so.  

Thanks for your time,
Kieran Hiller

<img width="570" alt="grod axis circle" src="https://user-images.githubusercontent.com/103440219/162780852-4b9b66e6-d149-49c0-8fec-70a4e97b8e71.png">
r

**Comment by kilir** [comment]
Hi,
you are plotting the misorientation axis (misorientation of all indexed orientations in your map with (presumably) a reference orientation) in specimen coordinates. This is a vector3d and may cover the entire (half)spehere- The radius of the sphere you see in the projection is 90 degree.
Hope this helps.
Rüdiger

**Reply by kieranhiller** [reply]
Thanks Rudiger. I thought that was the case, however when inspecting the sphere it seems that the misorientstion axis is only, at maximum, 1.4 degrees away from the reference in either the X or y direction wrt to the specimen reference frame. These points appear at the edge of the projection where a misorientation of 90 degrees would be expected. Am I interpreting this wrong?

Kind regards,
Kieran

**Reply by kilir** [reply]
I'm not entirely sure what you mean by inspecting. Could you paste the code you got the number from?

Cheers,
Rüdiger

**Reply by kieranhiller** [reply]
For example when I click the points on the sphere, as shown in my image, I can see the deviation in degrees with respect to the x and y axes. Also, just from looking at the Euler angles there is no change close to 5 degrees around any axis from the reference nevermind 90 degrees. I hope this makes sense.

Thanks, 
Kieran

**Reply by kilir** [reply]
What you see on the data tool tip for a direction mapping or if you plot vector3d are x,y position projected on the plane we are looking at. v.rho and v.theta will give you the polar coordinates in radians.
Also, the magnitude in difference of Euler angles between two orientations is unrelated to the direction of the misorientation axis in specimen coordinates. 
Hope this helps.
Cheers,
Rüdiger
